### PR TITLE
Fixing chat for email username format

### PR DESCRIPTION
**Pull request by sayachi**

### CODE REVIEW

File: application/src/main/webapp/js/chat.js [hunk 1]
```diff
@@ -2403,7 +2403,9 @@ ChatApplication.prototype.loadRoom = function() {
 
     jqchat(".users-online").removeClass("accordion-active");
     if (this.isDesktopView()) {
-      var $targetUser = jqchat("#users-online-"+this.targetUser.replace(".", "-"));
+      var $targetUser = this.targetUser.replace(".", "-") ;
+      var element = document.getElementById($targetUser);
+      $targetUser = jqchat("#users-online-"+element);
```
Comment (thomasdelhomenie) [inline application/src/main/webapp/js/chat.js]: "element" is an object. Why is it concatenated in a string ?

File: application/src/main/webapp/js/chat.js [hunk 2]
```diff
@@ -2403,7 +2403,9 @@ ChatApplication.prototype.loadRoom = function() {
 
     jqchat(".users-online").removeClass("accordion-active");
     if (this.isDesktopView()) {
-      var $targetUser = jqchat("#users-online-"+this.targetUser.replace(".", "-"));
+      var $targetUser = this.targetUser.replace(".", "-") ;
+      var element = document.getElementById($targetUser);
```
Comment (thomasdelhomenie) [inline application/src/main/webapp/js/chat.js]: Why not using jqchat instead of document.getElementById ?

Comment (sayachi) [inline application/src/main/webapp/js/chat.js]: When i passed the targetUser **directly** into jqchat parameters, there's a problem with the "@" character. It is **resolved** only when i used the **getElementById** method

Comment (thomasdelhomenie) [inline application/src/main/webapp/js/chat.js]: Escaping the @ does not work ?

Comment (sayachi) [inline application/src/main/webapp/js/chat.js]: It doesn't work.

File: application/src/main/webapp/js/chat.js [hunk 3]
```diff
@@ -2403,7 +2403,7 @@ ChatApplication.prototype.loadRoom = function() {
 
     jqchat(".users-online").removeClass("accordion-active");
     if (this.isDesktopView()) {
-      var $targetUser = jqchat("#users-online-"+this.targetUser.replace(".", "-"));
+      var $targetUser = jqchat("#users-online-"+this.targetUser.replace(/[.@]/g, "-"));
```
Comment (thomasdelhomenie) [inline application/src/main/webapp/js/chat.js]: Any reason why we do not simply keep the username as is and escape the @ in the jquery selector ( @ -> \\@ ) ? Actually I have the same question for the dot.

File: application/src/main/webapp/js/chat.js (hunk 2 above)
Comment (thomasdelhomenie) [inline application/src/main/webapp/js/chat.js]: Ah, I did not see this comment before posting my new comment.
I did a test and it seems to work : http://jsfiddle.net/20n5spfq/

**Pull request closed without merging**